content/cloud/iot-cloud/tutorials/02.technical-reference/iot-cloud-tech-ref.md

```diff
@@ -249,42 +249,80 @@ Declared as `CloudDimmedLight x;`
-| ----------| -------------------| -------------------------------| ------------------------------|
-| Switch    | `bool`             | `x.getSwitch()`                | `x.setSwitch()`               |
-| Hue       | `float` (0-360)    | `x.getHue()`                   | `x.setHue()`                  |
-| Saturation| `float` (0-100)    | `x.getSaturation()`            | `x.setSaturation()`           |
-| Brightness| `float` (0-100)    | `x.getBrightness()`            | `x.setBrightness()`           |
+| Property   | Type            | Read value          | Set value           |
+| ---------- | --------------- | ------------------- | ------------------- |
+| Switch     | `bool`          | `x.getSwitch()`     | `x.setSwitch()`     |
+| Hue        | `float` (0-360) | `x.getHue()`        | `x.setHue()`        |
+| Saturation | `float` (0-100) | `x.getSaturation()` | `x.setSaturation()` |
+| Brightness | `float` (0-100) | `x.getBrightness()` | `x.setBrightness()` |
 
 #### CloudColor
 
-Declared as `CloudColor x;`
+Declared as `CloudColor x;`.
+
+To read the Color values, we can use the following method `Color colorValues = x.getValue();`. This will assign the hue, saturation, and brightness values to the `colorValues` variable.
 
-To read the Color values, we can use the following method `Color colorValues = x.getValue();`. This will assign the hue, saturation, and brightness values to the colorValues variable. If we want too access the values individually we can use `Serial.println(colorValues.hue)`, `Serial.println(colorValues.sat)`, and `Serial.println(colorValues.bri)`.
+| Property   | Type    | Read value        | Set value                              |
+| ---------- | ------- | ----------------- | -------------------------------------- |
+| Hue        | `float` | `colorValues.hue` | `x = Color(hue,saturation,brightness)` |
+| Saturation | `float` | `colorValues.sat` | `x = Color(hue,saturation,brightness)` |
+| Brightness | `float` | `colorValues.bri` | `x = Color(hue,saturation,brightness)` |
 
 To set the color, we can assign the CloudColor variable directly to float variables `x = {hue,saturation,brightness}`, or using the method ` x = Color(hue,saturation,brightness)`.
 
 #### CloudLocation
 
-Declared as `CloudLocation x;`
+Declared as `CloudLocation x;`.
 
-To read the location values, we can use the following method `Location coordinates = x.getValue();`. This will assign the longitude and latitude values to the coordinates variable. If we want too access the values individually we can use `Serial.println(coordinates.lat)` and `Serial.println(coordinates.lat)`.
+To read the location values, we can use the following method `Location coordinates = x.getValue();`. This will assign the longitude and latitude values to the coordinates variable. If we want too access the values individually we can use `Serial.println(coordinates.lat)` and `Serial.println(coordinates.lon)`.
```

Comment:
Can we add somewhere that the format of `lat` and `lon` is Decimal degrees (DD) (ex. 41.40338, 2.17403) 🙏  ?

Reply:
Sure thing, I just added a note in the section to highlight it: 

"The format of the `lat` and `lon` is in Decimal Degrees (DD), for example `41.40338`, `2.17403`."